### PR TITLE
test: Enhancements to WACI issuance BDD tests

### PR DESCRIPTION
**Pull request by DRK3**
- Attachments are now generated on the fly using pre-defined Credential Manifest, Credential Application, and Credential Fulfillment objects.
- Added constants to Credential Manifest package for attachment formats and contexts.

Signed-off-by: Derek Trider <Derek.Trider@securekey.com>

### CODE REVIEW

File: test/bdd/pkg/waci/waci_issuance_sdk_steps.go [hunk 1]
```diff
@@ -70,6 +81,7 @@ func (i *IssuanceSDKSteps) SetContext(ctx *context.BDDContext) {
 }
 
 // RegisterSteps registers the BDD test steps on the suite.
+// Note that VC proofs are not checked in this test suite.
```
Comment (DRK3) [inline test/bdd/pkg/waci/waci_issuance_sdk_steps.go:84]: Should I also check VC proofs? (if so, I can do that in my next PR) @sudeshrshetty @rolsonquadras

File: test/bdd/pkg/waci/waci_issuance_sdk_steps.go [hunk 2]
```diff
@@ -699,6 +674,275 @@ type prop interface {
...
+		ID:        uuid.New().String(),
+		MediaType: "application/json",
+		Format:    cm.CredentialFulfillmentAttachmentFormat,
+		Data:      decorator.AttachmentData{JSON: attachmentData},
+	}
+
+	return &credentialFulfillmentAttachment, nil
+}
+
+func generateCredentialFulfillmentProof() map[string]string {
```
Comment (DRK3) [inline test/bdd/pkg/waci/waci_issuance_sdk_steps.go:922]: Not sure if this is the best way to do this... let me know if there's something else I should do instead. Also let me know if `did:orb` is ok here.

File: test/bdd/pkg/waci/credential_application_drivers_license.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (Moopli) [inline test/bdd/pkg/waci/credential_application_drivers_license.json:1]: is it possible to move these files (and the others in the `waci` folder) to a `testdata` subdirectory like elsewhere in `test/bdd/pkg` ?

Comment (DRK3) [reply test/bdd/pkg/waci/credential_application_drivers_license.json]: Done!

**Review by Moopli — APPROVED** [review]

File: test/bdd/pkg/waci/waci_issuance_sdk_steps.go (hunk 1 above)
Comment (sudeshrshetty) [reply test/bdd/pkg/waci/waci_issuance_sdk_steps.go:84]: that's ok, but nice to have feature. If it can be quick then do it.

**Review by sudeshrshetty — APPROVED** [review]

**Pull request merged**